ambari-logsearch/pom.xml

```diff
@@ -131,6 +131,28 @@
       </plugins>
     </pluginManagement>
     <plugins>
+       <plugin>
+         <groupId>org.vafer</groupId>
+         <artifactId>jdeb</artifactId>
+         <version>1.0.1</version>
```

Comment:
can you use version 1.4 for infra and logsearch?
note: just let you know, after 2.7.1 the ambari repo will be splitted into ambari, ambari-metrics,ambari-logsearch, ambari-infra, so those wont be built together (so possibly these changes wont be required for trunk)

Reply:
OK sure I will fix it and re-push the patch, Thanks for the suggestion @oleewere

Reply:
@oleewere my apologies for delay.  As you have suggested I have modified and given pull request only for v2.7 branch https://github.com/apache/ambari/pull/1840
Should we close this pull request  because it is for trunk ?

Reply:
@oleewere ok Thanks,  I have updated the pull request.  Now the patch contains jdeb v1.4 for ambari-logsearch and ambari-infra